guide/english/certifications/coding-interview-prep/project-euler/problem-2-even-fibonacci-numbers/index.md

```diff
@@ -33,7 +33,7 @@ function fiboEvenSum(n) {
 ```js
 // We use memoization technique to save ith fibonacci number to the fib array 
 function fiboEvenSum(n){
```

Comment:
Could you change this function to the following? I found that it currently returns 43 rather than 44:

```js
function fiboEvenSum(n){
  const fib = [1, 1, 2]; 
  let sumEven = fib[2];
  function fibonacci(n){
    if (n <= 1) return fib[n]; // base condition
    else if (fib[n]) return fib[n]; // if the number exists in the array we cache it and return
    else {
      fib[n] = fibonacci(n-1) + fibonacci(n-2); // otherwise calculcate and save it to the array
      if (fib[n]%2 == 0) sumEven+=fib[n];  //if the number is even, add it to the sumEven variable
      return fib[n];
    }
  }
  fibonacci(n); // run the recursive function
  return sumEven;
}
```

Reply:
@scissorsneedfoodtoo yeah, the `let sumEven = fib[2]` instead of fib[1], Done!